templates/cluster-template-powervs.yaml

```diff
@@ -187,7 +187,6 @@ spec:
         bootstrapToken:
           apiServerEndpoint: ${IBMPOWERVS_VIP}:${API_SERVER_PORT:=6443}
           caCertHashes: []
-          token: ""
```

Comment:
error:
`The KubeadmControlPlane "ibm-powervs-1-control-plane" is invalid: spec.kubeadmConfigSpec.joinConfiguration.discovery.bootstrapToken.token: Invalid value: "": spec.kubeadmConfigSpec.joinConfiguration.discovery.bootstrapToken.token in body should be at least 1 chars long`